erroranalysis/erroranalysis/_internal/constants.py

```diff
@@ -153,7 +153,10 @@ class ErrorCorrelationMethods(str, Enum):
     Metrics.ERROR_RATE: 'Error rate',
     Metrics.FALSE_POSITIVE_RATE: 'False positive rate',
     Metrics.FALSE_NEGATIVE_RATE: 'False negative rate',
-    Metrics.SELECTION_RATE: 'Selection rate'
+    Metrics.SELECTION_RATE: 'Selection rate',
+    Metrics.MEAN_AVERAGE_PRECISION: 'Mean average precision',
```

Comment:
I'm a bit confused why these are still added here since you added only error rate now, but maybe its fine